src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -438,6 +438,13 @@ case class IndexLogEntry(
     properties: Map[String, String])
     extends LogEntry(IndexLogEntry.VERSION) {
 
+  /*
+   * Include the project version in the index log entry file.
+   * This may help prevent in mismatching of project version and index
+   * log entry files.
+   */
+  val projectVersion: String = BuildInfo.version
```

Comment:
Could you add this into `properties`? to avoid breaking changes.
Thanks!

Reply:
I put it in the properties of IndexLogEntry.